Assets/GothicVR/Scripts/Globals/Hero.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using GVR.Caches;
+using GVR.Properties;
+using UnityEngine.SceneManagement;
+using ZenKit.Daedalus;
+
+namespace GVR.Globals
+{
+    public static class Hero
```

Comment:
```suggestion
    public static class NpcHelper
```

Isn't it more like a Helper class to set hero? For these two methods you can leverage NpcHelper.cs